kyo-caliban/src/main/scala/kyo/resolvers.scala

```diff
@@ -0,0 +1,74 @@
+package kyo
+
+import caliban.*
+import caliban.interop.tapir.*
+import caliban.interop.tapir.TapirAdapter.*
+import kyo.internal.KyoSttpMonad
+import scala.concurrent.ExecutionContext
+import sttp.monad.Canceler
+import sttp.tapir.Codec.JsonCodec
+import sttp.tapir.Endpoint
+import sttp.tapir.capabilities.NoStreams
+import sttp.tapir.server.ServerEndpoint
+import sttp.tapir.server.netty.*
+import zio.*
+import zio.stream.ZStream
+
+opaque type Resolvers = Aborts[CalibanError] & ZIOs
+
+object Resolvers:
+
+    private given StreamConstructor[Nothing] =
+        (_: ZStream[Any, Throwable, Byte]) => throw new Throwable("Streaming is not supported")
```

Comment:
I think it'd require `inline given` as well

Reply:
But we don't want a compile error here, we need to provide a `StreamConstructor` for `NoStreams`, which is the only stream implementation we can use now. Once streaming is supported by `NettyKyoServer`, this can be removed.